.github/workflows/format.yml

```diff
@@ -6,26 +6,21 @@ jobs:
   prettier:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.ref }}
       - name: Install dependencies
         run: npm ci
-      - name: Cache node_modules
-        uses: actions/cache@v3
+      - uses: actions/setup-node@v3
         with:
-          path: |
-            ./node_modules
-          key: npm-workspace-cache-${{ runner.os }}-${{ hashFiles('./package-lock.json') }}
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
+          node-version: 20
```

Comment:
I'd rather update node versions in its own PR. Ideally I'd rather see us test against a matrix of supported versions, but if we're only doing one, I think it's probably less risky to keep it at "the earliest version we support".